lib/mix/tasks/phx.gen.html.ex

```diff
@@ -176,13 +176,13 @@ defmodule Mix.Tasks.Phx.Gen.Html do
       {key, :text} ->
         {label(key), ~s(<%= textarea f, #{inspect(key)} %>), error(key)}
       {key, :date} ->
-        {label(key), ~s(<%= date_select f, #{inspect(key)} %>), error(key)}
+        {label(key), ~s(<%= date_select f, #{inspect(key)}, year: [options: 2010..#{Date.utc_today().year}] %>), error(key)}
```

Comment:
Since params is using dates from `2010` I had to choose either update there or here or the date select will not have `2010` as an option. The issue updating the params is that tests will break after some years, that is why I'd rather keep this change here.

Reply:
Interesting. I prefer to change the value we generate for the tests then, make sure that it always mirrors Date.utc_today. WDYT?

Reply:
To be more precise, I would change `type_to_default` to return `Date.add(Date.utc_today, -1, :days)` for create and `Date.utc_today()` for update. We need to do similar to naive_datetime and friends too. ALthough for NaiveDateTime we can do -1 hour. Thoughts?

Reply:
Make more sense haha 👍 

Just updated. For `NaiveDatetime` and `Datetime` I am doing `-100_000` which goes backwards one day, hope that is fine 🙈